osu.Game/Users/Drawables/DrawableAvatar.cs

```diff
@@ -30,7 +34,10 @@ public DrawableAvatar(User user = null)
         [BackgroundDependencyLoader]
         private void load(LargeTextureStore textures)
         {
-            if (user != null && user.Id > 1)
+            string avatarUrl = user?.AvatarUrl;
+            if (api != null && avatarUrl != null)
+                Texture = textures.Get(avatarUrl.StartsWith('/') ? $"{api.WebsiteRootUrl}{avatarUrl}" : avatarUrl);
```

Comment:
where does the '/' requirement come from?

Reply:
Its on OsuGameBase#OpenUrlExternally

Reply:
Not sure I follow. Sure, that logic is in that method, but where does this ever occur for an avatar? ie. which source is providing relative avatar URLs currently?

Reply:
as i mentioned at description at pr, default avatar_ url seems to give relative url one

Reply:
Coming from the API?

Reply:
yes, probably

Reply:
I've brought this up in an osu-web issue. Probably best we standardise this rather than working around it.

Reply:
I will revert this on the assumption that it will have a full URL.